src/components/Navbar/NavBar.tsx

```diff
@@ -189,21 +195,33 @@ const LanguageContainer = styled.div`
 `;
 const LanguageButton = styled.div`
   margin: 0;
+  transition: 0.1s;
   color: #9e9e9e;
   font-size: 14px;
   cursor: pointer;
   font-weight: 200;
+  :hover {
+    color: #a7182d;
+  }
 `;
 const ReactNavLink = styled(Link)`
   text-decoration: none;
   color: black;
+  transition: 0.2s;
   ${(props: CompactProps) =>
-    props.compact &&
+    props.compact === 'true' &&
```

Comment:
any reason is a string and not a boolean?

Reply:
chrome throws a console warning saying it should be a string if youre using props to change css through styled components